code/game/machinery/doors/door.dm

```diff
@@ -102,6 +102,10 @@ var/list/all_doors = list()
 				playsound(src.loc, 'sound/machines/denied.ogg', 50, 1)
 				door_animate("deny")
 
+/obj/machinery/door/bumped_by_firebird(var/obj/structure/bed/chair/vehicle/wizmobile/W)
+	W.forceMove(get_step(W,W.dir))//Firebird doesn't wait for no slowpoke door to fully open before dashing through!
```

Comment:
This is from #4379 and I really don't like it because I know it's not proper
I'm unsure how else to implement it though